index.js

```diff
@@ -0,0 +1,8 @@
+alert("hi");
+var critical = false;
+if( 1 === 1 && 1 < 2 ) {
```

Comment:
BUG: Correct one of the identical sub-expressions on both sides of operator &quot;===&quot;